tests/Tests/100__deploy/130__operators/130__rhods_operator/135__rhods_operator_logs_verification.robot

```diff
@@ -23,7 +23,7 @@ Verify RHODS Operator log
    #Capture the logs based on containers
    ${val}        Run   oc logs --tail=1000000 ${data[0]['metadata']['name']} -n ${namespace} -c rhods-operator
    #To check if command has been suessfully executed and the logs has been captured
-   Run Keyword IF    len($val)==${0} or "error" in $val     FAIL   Either OC command has not been executed suessfully or Logs is not present
+   Run Keyword IF    len($val)==${0} or "error:" in $val     FAIL   Either OC command has not been executed suessfully or Logs is not present
```

Comment:
There is a typo in "successfully"

LGTM as a hotfix. If you want to improve it after maybe you can check the return status differently using the keyword  
 `Run Command` :

```
${result} =  Run Process    oc logs --tail=1000000 ${data[0]['metadata']['name']} -n ${namespace} -c rhods-operator
Should Be Equal As Integers  ${result.rc}  0
```
More info at  https://robotframework.org/robotframework/latest/libraries/Process.html

Reply:
Done...For now i have kept the same ..will see in future if we need to use Process library